### PR TITLE
Redesign the JavaScript driver page

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-getting-started-339.surge.sh

File: modules/ROOT/pages/languages-guides/neo4j-javascript.adoc [hunk 1]
```diff
@@ -12,60 +12,29 @@
...
-The Neo4j JavaScript driver is *officially supported* by Neo4j and connects to the database using the binary protocol.
-It aims to be minimal, while being idiomatic to JavaScript, allowing you to subscribe to a stream of responses, errors and completion events.
-
-[source, shell, subs=attributes,specialcharacters]
-----
-npm install neo4j-driver
-----
+The Neo4j JavaScript driver connects to the database using the binary protocol Bolt.
+It aims to be minimal while being idiomatic to JavaScript, allowing you to subscribe to a stream of responses, errors, and completion events.
+You can use the official driver for JavaScript or <<js-http-endpoint, HTTP API>>.
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/languages-guides/neo4j-javascript.adoc:28]:
```suggestion
```
I don't see a reason to mention the http api here.

File: modules/ROOT/pages/languages-guides/neo4j-javascript.adoc [hunk 2]
```diff
@@ -132,40 +100,22 @@ You will find the implementations for all drivers as https://github.com/neo4j-ex
...
-  }],
- "errors":[]}
-----
+== Using the HTTP endpoint directly
+
+You can use Neo4j HTTP API to send Cypher queries to and receive responses from Neo4j.
+
+The endpoint protocol and formats are explained in detail in the link:https://neo4j.com/docs/http-api/current/[Neo4j HTTP API Manual^].
+
+It enables you do to much more, for example sending many statements per request or keeping transactions open across multiple requests.
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/languages-guides/neo4j-javascript.adoc:109]:
```suggestion
```
I'd get rid of all this HTTP section.

**Review by stefano-ottolenghi — APPROVED** [review]

File: modules/ROOT/pages/languages-guides/neo4j-javascript.adoc (hunk 2 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/languages-guides/neo4j-javascript.adoc]: Thank you for reviewing! I think, it's a good idea

**Pull request merged**